example/Example.WeatherService/Program.cs

```diff
@@ -2,10 +2,7 @@
 using Serilog.Templates.Themes;
 using SerilogTracing;
 using SerilogTracing.Expressions;
-using SerilogTracing.Instrumentation.AspNetCore;
 using SerilogTracing.Sinks.OpenTelemetry;
```

Comment:
This remains because the `OtlpProtocol` enum is still under the nested namespace. The general rule is that extensions go in the root `SerilogTracing` so that they're easy to discover, and everything else goes where you'd expect.